### PR TITLE
Minor cleanup to the Camel rest options

### PR DESCRIPTION
**Pull request by orpiske**

### CODE REVIEW

**Comment by orpiske** [comment]
This is a draft, but please wait until we are OK to merge Camel 4 changes.

File: docs/user-manual/modules/ROOT/pages/camel-3x-upgrade-guide-3_21.adoc
Comment (orpiske) [inline docs/user-manual/modules/ROOT/pages/camel-3x-upgrade-guide-3_21.adoc:97]: Placeholder. I'll move once we have the 4x migration file.

**Comment by orpiske** [comment]
Actually, I think this one can go to 3.21 as well

**Review by davsclaus — APPROVED** [review]

**Pull request merged**
